app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command) != 0:
+        cp, old_name, new_name = command.split()
+        if old_name != new_name and cp == "cp":
+            with open(old_name, "r") as file_in:
+                with open(new_name, "w") as file_out:
```

Comment:
```suggestion
            with (
                open(old_name, "r") as file_in,
                open(new_name, "w") as file_out
            ):
```

Reply:
when I make this way I have invalid syntax error. If I use slash everything ok. 
That's why I changed with 2 WITH OPEN

Reply:
Pls, use Python 3.10 to avoid this issue in a future